src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -2871,6 +2871,12 @@
   <data name="ReleaseNotes" xml:space="preserve">
     <value>Release Notes</value>
   </data>
+  <data name="CannotCreateShortcutDialogTitle" xml:space="preserve">
+    <value>This action requires administrator privileges.</value>
+  </data>
+  <data name="CannotCreateShortcutDialogMessage" xml:space="preserve">
+    <value>Please run Files as administrator or create the shortcut on the Desktop instead.</value>
```

Comment:
```suggestion
    <value>Please run Files as administrator or create the shortcut somewhere else instead.</value>
```

Reply:
@yaira2 at the moment if the creation of the shortcut fails and the user hits `Create` the shortcut will be placed on the Desktop

Reply:
Thanks! I edited the strings to clarify that.